docs/src/example.md

```diff
@@ -93,44 +88,145 @@ So far, we only added some parameters that had no influence at first in solving
-julia> isapprox.(MOI.get(optimizer, MOI.VariablePrimal(), x[2]), 4/3, atol = 1e-4)
+isapprox.(MOI.get(optimizer, MOI.ObjectiveValue()), 28/3, atol = 1e-4)
+isapprox.(MOI.get(optimizer, MOI.VariablePrimal(), x[1]), 4/3, atol = 1e-4)
+isapprox.(MOI.get(optimizer, MOI.VariablePrimal(), x[2]), 4/3, atol = 1e-4)
 ```
 
 Although we changed the parameter values, we didn't optimize the model yet. Thus, **to apply the parameters' changes, the model must be optimized again**:
 
 ```julia
-julia> MOI.optimize!(optimizer)
+MOI.optimize!(optimizer)
 ```
 
 The `MOI.optimize!()` function handles the necessary updates, properly fowarding the new outer model (`POI` model) additions to the inner model (`MOI` model) which will be handled by the solver. Now we can assess the updated optimized information:
 
 ```julia
-julia> @test isapprox.(MOI.get(optimizer, MOI.ObjectiveValue()), 3.0, atol = 1e-4)
-julia> @test MOI.get.(optimizer, MOI.VariablePrimal(), x) == [0.0, 1.0]
+isapprox.(MOI.get(optimizer, MOI.ObjectiveValue()), 3.0, atol = 1e-4)
+MOI.get.(optimizer, MOI.VariablePrimal(), x) == [0.0, 1.0]
 ```
 
 If we update the parameter `w`, associated to the objective function, we are simply adding a constant to it. Notice how the new objective function is precisely equal to the previous one plus the new value of `w`. In addition, as we didn't update the feasible set, the optimized decision variables remain the same.
 
 ```julia
-julia> MOI.set(optimizer, MOI.ConstraintSet(), cw, POI.Parameter(2.0))
+MOI.set(optimizer, MOI.ConstraintSet(), cw, POI.Parameter(2.0))
 # Once again, the model must be optimized to incorporate the changes
-julia> MOI.optimize!(optimizer)
+MOI.optimize!(optimizer)
 # Only the objective function value changes
-julia> @test isapprox.(MOI.get(optimizer, MOI.ObjectiveValue()), 7.0, atol = 1e-4)
-julia> @test MOI.get.(optimizer, MOI.VariablePrimal(), x) == [0.0, 1.0]
+isapprox.(MOI.get(optimizer, MOI.ObjectiveValue()), 7.0, atol = 1e-4)
+MOI.get.(optimizer, MOI.VariablePrimal(), x) == [0.0, 1.0]
+```
+
+## JuMP Example
+
+Lets write a setep-by-step example of `POI` usage at the JuMP level.
+
+First, we declare a `Model` on top of a `Optimizer` of an underlying solver. In the example, we consider `HiGHS` as the underlying solver:
+
+```julia
+using HiGHS
+using JuMP
+
+using ParametricOptInterface
+const POI = ParametricOptInterface
+
+model = Model(() -> ParametricOptInterface.Optimizer(HiGHS.Optimizer()))
+```
+
+We declare the variable `x` as in a typical `JuMP` model:
+
+```julia
+@variable(model, x[i = 1:2] >= 0)
 ```
 
+Now, let's consider 3 parameters. Two of them, `y`, `z`, will be placed in the constraints and one, `w`, in the objective function. We'll start all three of them with a value equal to `0`:
 
+```julia
+@variable(model, y in ParametricOptInterface.Parameter(0))
+@variable(model, z in ParametricOptInterface.Parameter(0))
+@variable(model, w in ParametricOptInterface.Parameter(0))
```

Comment:
we will need another example with a few versions of the vectorial syntax:
```julia
@variable(model, w[i in 1:3] in ParametricOptInterface.Parameter(0))
```

```julia
@variable(model, w[i in 1:3] in ParametricOptInterface.Parameter(3 * i + 2))
```

```julia
@variable(model, w[i in 1:3] in ParametricOptInterface.Parameter.(ones(3)))
```

I don't even know if all these work, but we should try showing most cases that work.